app/model/ReminderNotice.scala

```diff
@@ -17,6 +17,7 @@
 package model
 
 import model.ProgressStatuses._
```

Comment:
Unrelated to the ticket, but i had fixed a couple of warnings, hoping it's ok here